code/index.html

```diff
@@ -8,11 +8,63 @@
   </head>
```

Comment:
Don't forget the meta tag ;) Also - change the title to something that fits your site
`<meta name="viewport" content="width=device-width, initial-scale=1">`